Fluid.Tests/StringFiltersTests.cs

```diff
@@ -300,7 +300,9 @@ public void TruncateWithCustomEllipsis(string input, int size, string ellipsis,
         [InlineData("The cat came back the very next day", 4, "The cat came back...")]
         [InlineData("The cat came back the very next day", 1, "The...")]
         [InlineData("The cat came back the very next day", 0, "...")]
-        [InlineData("The    cat came  back", 10, "The    cat came  back...")]
+        [InlineData("The cat came back the very next day", 8, "The cat came back the very next day...")]
```

Comment:
Can you confirm this is the behavior in Shopify?

Reply:
I just tried them in Liquid JS

Reply:
I know. LiquidJs is not bullet proof, I have files issues there when I found bugs.

Tests to add:
https://github.com/Shopify/liquid/blob/ff1c35b986451cca14d9da9b2ef772cbc9c65b24/test/integration/standard_filter_test.rb#L217-L234

And one that got removed I think and which was wrong, is to add multiple spaces, they should be reduced to a single single one, like a `String.Join(" ", String.Split(RemoveEmptyEntries))`

Reply:
I see the test wasn't removed, but the result it wrong.

Reply:
I see, I will check the ruby code tomorrow, coz I feel sleepy ;)